HacktoberfestProject.Web/Services/TableService.cs

```diff
@@ -15,20 +15,31 @@ namespace HacktoberfestProject.Web.Services
     public class TableService : ITableService
     {
         private readonly ITableContext _tableContext;
+        private readonly IGithubService _githubService;
 
-        public TableService(ITableContext tableContext)
+        public TableService(ITableContext tableContext, IGithubService githubService)

```

Comment:
Not entirely sure about the `TableService` needing another service (`GithubService`) injected. Somehow it feels like this last one is a separate service the application can use to talk to GitHub to do things like determine the status of a PR. I think it might be better to keep the responsibilities of the `TableService` clear. Or maybe that name isn't correct anymore and should now be a `TrackerEntityService`. 
We might need these services to be renamed to something like `Repository` and a new actual service added to enable the orchestration around one or more services working together. Although this _could_ also be in the `Controller`.

Do you have a preference on this @Layla-P ?

Reply:
@rickvdbosch  I understand your observations here. I work to the principal of make some thing work then enginer later once you know what all the moving parts are.